level07/src/FirstApp/AppM.hs

```diff
@@ -126,8 +114,8 @@ runAppM =
 -- throwError :: MonadError e m => e -> m a
 -- pure :: Applicative m => a -> m a
 --
-throwL
+throwLeft
```

Comment:
Is this a good name for this? Maybe liftEither?

Reply:
In my codebases  it's often called `rightOrThrow`, but I like `liftEither` too.